src/server/plugins/engine/components/DatePartsField.test.ts

```diff
@@ -12,7 +18,7 @@ describe('Date parts field', () => {
       schema: {}
     }
 
-    const underTest = new DatePartsField(def, {})
+    const underTest = new DatePartsField(def, {} as FormModel) // FormModel param not required for testing
```

Comment:
Can you leave out the `as` casting please? I've slowly been fixing things like this (pretending something is a different type)

Reply:
Ideally upgrade to one of the newer test files where we instantiate `new FormModel()`

E.g. Here's the one for email component:

https://github.com/DEFRA/forms-runner/blob/31e4713d105065272db5b15535d878332cc66e61/src/server/plugins/engine/components/EmailAddressField.test.ts#L19-L25

With a 100% valid `new EmailAddressField(def, formModel)` for each test:

https://github.com/DEFRA/forms-runner/blob/31e4713d105065272db5b15535d878332cc66e61/src/server/plugins/engine/components/EmailAddressField.test.ts#L32-L43

Reply:
Good timing, I just committed exactly that :). Thanks, done!